logviewer/api/handlers.go

```diff
@@ -57,6 +62,12 @@ func logHandler(w http.ResponseWriter, r *http.Request, containerID, fileName st
 		return
 	}
 	defer shouldClose(fout)
+
+	if addDownloadHeader {
+		basename := filepath.Base(fileName)
+		w.Header().Set("Content-Disposition", "attachment; filename="+basename)
```

Comment:
Is there any escaping necccessary? For example, if the filename has a ; in it?

Reply:
Uhhh maybe?
https://stackoverflow.com/questions/93551/how-to-encode-the-filename-parameter-of-content-disposition-header-in-http

But modern docs:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Disposition

Sounds like you just need quotes, but whoops, I didn't end the quote here!

But yea, sounds like it is a bit of a mess here, but it doesn't sound like I need to urlencode the name.

But it does sound like I need to escape quotemarks

Reply:
Fixed trailing quote and added a comment linking to the docs on this header